Resources/Prototypes/_Goobstation/Recipes/Lathes/research.yml

```
@@ -0,0 +1,15 @@
+# SPDX-License-Identifier: AGPL-3.0-or-later
+
+- type: latheRecipe
+  id: ExperimentalCircuitBoard
+  result: ExperimentalCircuitBoard
+  completetime: 7
+  materials:
+    Steel: 100
+    Glass: 500
+    Silver: 200
+    Gold: 200
+    Diamond: 100
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)